Assets/UI/Panels/citypanel.lua

```diff
@@ -86,18 +92,19 @@ local m_isShowingPanels :boolean= false;
 local m_pPlayer         :table  = nil;
 local m_primaryColor    :number = UI.GetColorValueFromHexLiteral(0xcafef00d);
 local m_secondaryColor  :number = UI.GetColorValueFromHexLiteral(0xf00d1ace);
-local m_CurrentPanelLine:number = 0;
 local m_kTutorialDisabledControls :table = nil;
```

Comment:
changed to match the order in the unmodified Firaxis citypanel.lua (so when comparing unmodified citypanel.lua to CQUI citypanel.lua, this does not appear as a difference in the two files).